synapseclient/models/project.py

```diff
@@ -365,27 +368,18 @@ async def get_async(
         """
         entity_id = await get_id(entity=self, synapse_client=synapse_client)
 
-        loop = asyncio.get_event_loop()
-        current_context = context.get_current()
-        entity = await loop.run_in_executor(
-            None,
-            lambda: run_and_attach_otel_context(
-                lambda: Synapse.get_client(synapse_client=synapse_client).get(
-                    entity=entity_id,
-                ),
-                current_context,
-            ),
+        await get_from_entity_factory(
+            entity_to_update=self,
```

Comment:
is this an incomplete attribute?

Reply:
no, it's intentional. This argument is `entity_to_update: Union["Project", "File", "Folder"] = None`. So in this case since it used within one of those classes I pass `self` into this so updates occur on the class instance passed in.